src/pybind/mgr/sendemail/module.py

```diff
@@ -0,0 +1,50 @@
+
+"""
+A hello world module
+
+See doc/mgr/hello.rst for more info.
+"""
+from mgr_module import MgrModule
+import smtplib
+from email.mime.text import MIMEText
+
+
+
+class SendEmail(MgrModule):
+
+    COMMANDS = [
+        {
+            "cmd": "sendemail ",       
+            "desc": "send email",
+            "perm": "rw"
+        },
+    ]
+
+    def handle_command(self, inbuf, cmd):
+
+        self.log.error("hello_world_error")
+        smtp_ssl_host = 'smtp.gmail.com'  # smtp.mail.yahoo.com
+	smtp_ssl_port = 465
+	sender = 'add Your Email here'
+	password = 'add Your Password here'
+	target = 'add the target email here'
+
+	msg = MIMEText('Hi, how are you today?')   
```

Comment:
Try making the body a dump of self.get('health') (e.g., 'heatlh status:\n%s' % self.get('health'))